resources/views/danabus/routes/show.blade.php

```diff
@@ -0,0 +1,48 @@
+@extends('templates.danabus.master')
+
+@section('content')
+  <div class="section secondary-section " id="portfolio">
+    <div class="triangle"></div>
+    <div class="container">
+      <div class=" title">
+        <h1>{{trans('admin_routes.routes')}} : {{$route->name}} </h1>  
+      </div>
+      <div id="single-project">
+        <div class="newcontent">
+          {!! trans('admin_routes.routes') !!} :{{$route->name}}<br>
+          {!! trans('admin_routes.forwardtrip') !!}: 
+          @foreach($route->forwardDirections as $forwardDirection) 
```

Comment:
it's difficult to understand this code, pls make it clear. Separate foreach loop and print out value